lib/matplotlib/axes/_axes.py

```diff
@@ -7880,14 +7880,14 @@ def matshow(self, Z, **kwargs):
     @_preprocess_data(replace_names=["dataset"])
     def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
                    showmeans=False, showextrema=True, showmedians=False,
-                   points=100, bw_method=None):
+                   percentiles=None, points=100, bw_method=None):
```

Comment:
Name must be changed to `quantiles`. The implementation correctly implements quantile-behavior.

Reply:
Code and documentation updated. Please have a check!